qiskit_aer/primitives/estimator_v2.py

```diff
@@ -70,12 +70,20 @@ def __init__(
                 the runtime options (``run_options``).
         """
         self._options = Options(**options) if options else Options()
-        method = "density_matrix" if "noise_model" in self.options.backend_options else "automatic"
-        self._backend = AerSimulator(method=method, **self.options.backend_options)
-
-    def from_backend(self, backend, **options):
-        """use external backend"""
-        self._backend.from_backend(backend, **options)
+        if "method" not in self.options.backend_options:
+            method = (
+                "density_matrix" if "noise_model" in self.options.backend_options else "automatic"
+            )
```

Comment:
Why is this line necessary? Shouldn't we just let AerSimulator set the method if it's not specified? It should default to `automatic`, which should be smart enough to choose `density_matrix` if `noise_model` is specified.

Reply:
I removed this so now `method=automatic` is set by default and #2117 is fixed